java/core/src/test/org/apache/orc/TestColumnStatistics.java

```diff
@@ -37,7 +37,6 @@
 import java.sql.Timestamp;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.Arrays;
```

Comment:
ditto.

Reply:
Same as above.

Reply:
Usually, it's better to clean up this if you change the actual code in this file.

Reply:
@dongjoon-hyun, the changes are undone.